github/i18n/en.json

```diff
@@ -0,0 +1,3 @@
+{
+    "cmd_description": "Easily track and manage Github directly with this app."
```

Comment:
@Prince-Mendiratta  I think we can do something more descriptive.

`GitHub APP: Search and share Issues and Pull Request, Subscribe to Repository Events, create New Issues, Review and Merge Pull Requests and do much more right from Rocket.Chat.`

Reply:
Agreed, we can make it shorter.

Reply:
GitHub App: Search, Share, Subscribe GitHub Resources and do much more from RocketChat.

Reply:
@Prince-Mendiratta what do you think ? open to suggestions :)

Reply:
@Prince-Mendiratta  Seems good. Can you please post a screenshot of how it looks ?

Reply:
@Prince-Mendiratta  yeah ! seems good. Please do the changes in Code